core/runtime/impl/storage_wasm_provider.hpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_RUNTIME_STORAGE_WASM_PROVIDER_HPP
+#define KAGOME_CORE_RUNTIME_STORAGE_WASM_PROVIDER_HPP
+
+#include "runtime/wasm_provider.hpp"
+
+#include "storage/trie/trie_db.hpp"
+
+namespace kagome::runtime {
+
+  // key for accessing runtime from storage(hex representation of ":code")
+  inline const common::Buffer runtime_key =
```

Comment:
```suggestion
  inline const common::Buffer kRuntimeKey =
```

Reply:
this is a constant, we can use k-snake case.